manim/animation/transform.py

```diff
@@ -46,6 +46,35 @@
 
 
 class Transform(Animation):
+    """A Transform transforms a Mobject into a target Mobject.
+
+    Parameters
+    ----------
+    mobject
+        The Mobject to be transformed. The Mobject will be mutated into the target.
+    target_mobject
+        The Mobject that will be transformed into. Will become invisible and replaced by mobject.
+    path_func
+        A function defining the path that the mobject will take to target_mobject,
+        see `manim.utils.paths`.
+    path_arc
+        The arc (in radians) that the mobject will take to reach target if using a circular path
+        arc, see `path_arc_centers`.
+    path_arc_axis
+        The axis to rotate along if using a circular path arc, see `path_arc_centers`.
+    path_arc_centers
+        The center of the circle used in the Transform's radial path.
```

Comment:
This parameter is still confusing to me; at the very least it is named terribly. Oh well.

```suggestion
        The center of the circular arcs along which the points of ``mobject`` are
        moved by the transformation.
```